### DISCUSSION TITLE
suggestions breaking?

### DISCUSSION THREAD

**Discussion by jonkelling** [community/community#72622] [Copilot Conversations]
### Select Topic Area

Bug

### Body

<img width="451" alt="copilot3" src="https://github.com/community/community/assets/7707676/aa6bffac-e958-48ee-9ed9-4d346ae6e810">
<img width="485" alt="copilot2" src="https://github.com/community/community/assets/7707676/a53812ee-42af-40d1-902d-d186baee4b40">
<img width="487" alt="copilot1" src="https://github.com/community/community/assets/7707676/c23c884e-74ef-4525-8e41-0ed57afd0ff2">

Edit: seems to be a duplicate of https://github.com/orgs/community/discussions/72620

**Comment by weslley75** [comment]
Same here
![Screenshot from 2023-10-26 10-23-12](https://github.com/community/community/assets/7125447/5da26950-d44f-41b6-8f39-82c67d9a18ba)

**Comment by jonkelling** [comment]
seems to be working again
